star-ros2/src/star_perception/star_perception/_hailo_worker.py

```
@@ -0,0 +1,196 @@
+        hef = HEF(str(self.hef_path))
+        self.vdevice = VDevice()
+        cp = ConfigureParams.create_from_hef(
+            hef=hef, interface=HailoStreamInterface.PCIe
+        )
+        self._ng = self.vdevice.configure(hef, cp)[0]
+        self._in_name = hef.get_input_vstream_infos()[0].name
+        self._out_name = hef.get_output_vstream_infos()[0].name
+        self._ip = InputVStreamParams.make(self._ng, format_type=FormatType.UINT8)
+        self._op = OutputVStreamParams.make(self._ng, format_type=FormatType.FLOAT32)
+
+    def infer(self, canvas_hwc: np.ndarray):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `infer`

(ANN202)